.github/workflows/assign-reviewers.yml

```diff
@@ -12,4 +12,6 @@ jobs:
   assign-reviewers:
     runs-on: ubuntu-latest
     steps:
-      - uses: dyladan/component-owners@main
+      - uses: open-telemetry/assign-reviewers-action@main
+        with:
+          config-file: .github/component_owners.yml
```

Comment:
@dyladan I like the language of "component owners" so kept that part intact, e.g. https://github.com/open-telemetry/opentelemetry-java-contrib/tree/main/maven-extension#component-owners

Reply:
I also like owners better. I changed the name in order to better describe the actual behavior before, but I think we're going to extend it to do additional things aside from reviewer assignment. Maybe we should consider changing it back before too many people switch to the new repo?

Reply:
The file name and action name both use "assign-reviewers". The language should at least be consistent within the repo?